java/api/src/main/java/com/exadel/frs/core/trainservice/filter/SecurityValidationFilter.java

```diff
@@ -138,7 +138,7 @@ private void buildException(final HttpServletResponse response, final ResponseEn
     }
 
     private ModelType getModelTypeByUrl(String url) {
-        if (url.endsWith("/detection")) {
+        if (url.endsWith("/detect")) {
             return DETECTION;
```

Comment:
BTW we can simplify this code, because we have unique contexts:
/recognition
/detection
/verification
So we can do something like:
url.contains(API_V1 + "/recognition")
url.contains(API_V1 + "/detection")
url.contains(API_V1 + "/verification")

Reply:
Simplified as described